fuse/opcode.go

```diff
@@ -88,7 +88,7 @@ func doInit(server *Server, req *request) {
 
 	server.reqMu.Lock()
 	server.kernelSettings = *input
-	server.kernelSettings.Flags = input.Flags & (CAP_ASYNC_READ | CAP_BIG_WRITES | CAP_FILE_OPS |
+	server.kernelSettings.Flags = input.Flags & (CAP_ASYNC_READ | CAP_BIG_WRITES | CAP_FILE_OPS | CAP_ATOMIC_O_TRUNC |
```

Comment:
Not all the file systems implement this feature, it's better to have a config for it.